CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 - #171 – Updated React to 0.14. React is along with other dependencies are now required as peerDependencies.
 - #261 - Added Radio Button component to the Form component.
 - #262 – Change Checkbox component to extend Radio Button component and reuse functionality. Functionality of FieldCheckbox and FieldCheckboxMultiple is now both contained in FieldCheckbox.
+- #276 - Change Form updating to delete fields that are no longer part of the definition.
```

Comment:
This was added the wrong place. I see some of the ones above. Created a PR for it here: https://github.com/mesosphere/reactjs-components/pull/278